assets/js/src/common/set_from_address_modal.tsx

```diff
@@ -59,6 +90,20 @@ const showSetFromAddressModal = async () => {
         if (!address) {
           return;
         }
+        try {
+          await handleSave(address);
+          MailPoet.Modal.close();
+          MailPoet.Notice.success(getSuccessMessage());
```

Comment:
Why are you using this old notice? I thought they are deprecated since Amine created the new mechanism.

Reply:
Explained in https://github.com/mailpoet/mailpoet/pull/2680/files#r400026892. Simply, I don't know if there's a way to interact with the React-rendered tree from non-react code.